src/app/(dashboard)/explorer/constants.ts

```diff
@@ -1,5 +1,5 @@
 export const PAGE_SIZE = 24;
 export const ACTIVE_SECTION = {
-  yourGuilds: "your-guilds",
+  associatedGuilds: "associated-guilds",
```

Comment:
tbh, I wouldn't rename this, imo "Your guilds" is much more user friendly ;)

Reply:
Internally we started calling it like this since "your" implicates ownership, I agree that it's more user friendly.

Associated is a broader term where the logic behind it could be changed, without renaming. I'm merging this PR to move on, but we can figure out something for this.